src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -6,71 +6,87 @@
-    BABA_GAP("Ba-Ba Gap", (config) -> config.babaGapMessage(), "baba-gap.png"),
+    BABA_BANANA("Ba-Ba Banana", (config) -> config.babaBananaMessage(), "Who put that there?", "baba-banana.png"),
+    BABA_GAP("Ba-Ba Gap", (config) -> config.babaGapMessage(), "I'm going down!", "baba-gap.png"),
 
     // Wardens
-    WARDENS_P1_PYRAMID("Wardens P1 Pyramid", (config) -> config.wardensPyramidMessage(), "wardens-pyramid.png"),
-    WARDENS_P2_OBELISK("Wardens P2 Obelisk", (config) -> "", "wardens-obelisk.png"),
-    WARDENS_P2_DDR("Wardens P2 DDR", (config) -> config.wardensDDRMessage(), ""),
-    WARDENS_P2_WINDMILL("Wardens P2 Windmill", (config) -> config.wardensWindmillMessage(), ""),
-    WARDENS_P2_BOMBS("Wardens P2 Bombs", (config) -> config.wardensBombsMessage(), ""),
-    WARDENS_P2_BIND("Wardens P2 Bind", (config) -> config.wardensBindMessage(), "wardens-bind.png"),
+    WARDENS_P1_PYRAMID("Wardens P1 Pyramid", (config) -> config.wardensPyramidMessage(), "I'm disco-ing!",
+            "wardens-pyramid.png"),
+    WARDENS_P2_OBELISK("Wardens P2 Obelisk", (config) -> "", "", "wardens-obelisk.png"),
+    WARDENS_P2_DDR("Wardens P2 DDR", (config) -> config.wardensDDRMessage(), "I'm dancing!", ""),
+    WARDENS_P2_WINDMILL("Wardens P2 Windmill", (config) -> config.wardensWindmillMessage(), "I'm winded!", ""),
+    WARDENS_P2_BOMBS("Wardens P2 Bombs", (config) -> config.wardensBombsMessage(), "I'm getting bombed!", ""),
+    WARDENS_P2_BIND("Wardens P2 Bind", (config) -> config.wardensBindMessage(), "I'm in jail!", "wardens-bind.png"),
     WARDENS_P2_SPECIAL_PRAYER("Wardens P2 Special Prayer", (config) -> config.wardensPrayerMessage(),
-            "wardens-special-prayer.png"),
-    WARDENS_P3_EARTHQUAKE("Wardens P3 Slam", (config) -> config.wardensSlamMessage(), "wardens-earthquake.png"),
-    WARDENS_P3_AKKHA("Wardens P3 Akkha", (config) -> "", "wardens-akkha.png"),
-    WARDENS_P3_ZEBAK("Wardens P3 Zebak", (config) -> "", "wardens-zebak.png"),
-    WARDENS_P3_KEPHRI("Wardens P3 Kephri", (config) -> config.kephriBombMessage(), "wardens-kephri.png"),
-    WARDENS_P3_BABA("Wardens P3 Ba-Ba", (config) -> config.babaFallingBoulderMessage(), "wardens-baba.png"),
-    WARDENS_P3_LIGHTNING("Wardens P3 Lightning", (config) -> "", "wardens-lightning.png"), // Too noisy
+            "What even was that attack?", "wardens-special-prayer.png"),
+    WARDENS_P3_EARTHQUAKE("Wardens P3 Slam", (config) -> config.wardensSlamMessage(), "I'm tripping!",
+            "wardens-earthquake.png"),
+    WARDENS_P3_AKKHA("Wardens P3 Akkha", (config) -> "", "", "wardens-akkha.png"),
+    WARDENS_P3_ZEBAK("Wardens P3 Zebak", (config) -> "", "", "wardens-zebak.png"),
+    WARDENS_P3_KEPHRI("Wardens P3 Kephri", (config) -> config.kephriBombMessage(), "I'm exploding!",
+            "wardens-kephri.png"),
+    WARDENS_P3_BABA("Wardens P3 Ba-Ba", (config) -> config.babaFallingBoulderMessage(), "It's raining!",
```

Comment:
These  defaults should also probably reference their above counterparts:

`"I'm exploding!"` -> `KEPHRI_BOMB.getDefaultMessage()` and same for the baba one

Reply:
Dow.  Must've missed those